partials/extensions.html

```diff
@@ -0,0 +1,44 @@
+{% macro extension(obj, odd = false, indent = false) %}
+
+{% set extensions = obj | nonParserExtensions%} 
+{% if extensions.size > 0 %}
+      <p class="{% if indent %}pl-6{% endif %} mb-2 mt-4 text-sm font-bold text-gray-700">
+        Specification Extensions:
+      </p>
+      {% for extensionName, extensionValue in extensions %}
+      <div class="{% if odd %}bg-gray-100{% else %}bg-gray-200{% endif %} {% if indent %}pl-6{% endif %}">
```

Comment:
`{% if indent %}pl-6{% endif %}` is the only change I introduced to the code after extracting it from schema prop macro, to support conditional indent of spec extension data that sometimes is needed in the UI